sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/b2c/AADB2CAutoConfigurationTest.java

```diff
@@ -43,8 +45,12 @@ public void testPropertiesBean() {
             assertThat(properties.getReplyUrl()).isEqualTo(AADB2CConstants.TEST_REPLY_URL);
 
             final String signUpOrSignIn = properties.getUserFlows().getSignUpOrSignIn();
+            final Object prompt = properties.getAuthenticateAdditionalParameters().get(AADB2CConstants.PROMPT);
+            final String loginHint = String.valueOf(properties.getAuthenticateAdditionalParameters().get(AADB2CConstants.LOGIN_HINT));
 
             assertThat(signUpOrSignIn).isEqualTo(AADB2CConstants.TEST_SIGN_UP_OR_IN_NAME);
+            assertThat(prompt).isEqualTo(AADB2CConstants.TEST_PROMPT);
+            assertThat(loginHint).isEqualTo(AADB2CConstants.TEST_LOGIN_HINT);
```

Comment:
Hi, @han-gao , we should test method `AADB2CAuthorizationRequestResolver.resolve(...)` instead of only test `AADB2CProperties`

Reply:
Seems it already exist in AADB2CAuthorizationRequestResolverTest.java. 
OK, I'll merge it.